### PR TITLE
Add WACK testing for Windows Application Packaging Project with FullTrustProcess

### LINKED ISSUE #6175: Add WACK testing for Windows Application Packaging Project with FullTrustProcess
As a follow up to Issue #5768 (and Fix #6063) we should see if we can add this scenario to our WinUI WACK test suite. We currently have NugetPackageTestApp and AppThatUsesMUXIndirectly that we use for WACK validation. We should see if we can add a third app to cover the FullTrustProcess scenario.

### PR DESCRIPTION
**Pull request by kmahone**
This adds a new test app to the repo: UwpAppWithFullTrustProcess

This app consists of three projects: a UWP app, a full trust process and a packaging project. This scenario allows a UWP app to bundle a full trust process in its package and invoke the full trust process.

We build this app and run WACK tests on it to ensure that we don't introduce WACK certification issues to this scenario when we make changes to WinUI.

Related to #5768
Fixes #6175

### CODE REVIEW

File: build/AzurePipelinesTemplates/MUX-WACKTests-Job.yml [hunk 1]
```diff
@@ -68,6 +66,21 @@ jobs:
...
 
+  - task: PkgESWACKTests@12
+    displayName: Run WACK tests for UwpAppWithFullTrustProcess
+    env:
+      SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+    condition: succeededOrFailed()
+    inputs:
+      PackageFilePath: '$(appxPackagesPath)\UwpAppWithFullTrustProcess_package_Test\UwpAppWithFullTrustProcess_package.msix'
+  - task: CopyFiles@2
+    displayName: 'Copy results for AppThatUsesMUXIndirectly'
```
Comment (llongley) [inline build/AzurePipelinesTemplates/MUX-WACKTests-Job.yml:77]: Typo: should be UwpAppWithFullTrustProcess

File: test/MUXControlsReleaseTest/UwpAppWithFullTrustProcess/UwpAppWithFullTrustProcess/MainPage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    public sealed partial class MainPage : Page
+    {
+        public MainPage()
+        {
+            this.InitializeComponent();
+        }
+
+        private async void Button_Click(object sender, RoutedEventArgs e)
+        {
+			await FullTrustProcessLauncher.LaunchFullTrustProcessForCurrentAppAsync();
```
Comment (llongley) [inline test/MUXControlsReleaseTest/UwpAppWithFullTrustProcess/UwpAppWithFullTrustProcess/MainPage.xaml.cs:29]: Nit: tabs

**Review by llongley — APPROVED** [review]

File: build/AzurePipelinesTemplates/MUX-WACKTests-Job.yml (hunk 1 above)
Comment (kmahone) [reply build/AzurePipelinesTemplates/MUX-WACKTests-Job.yml]: Good catch.

File: test/MUXControlsReleaseTest/UwpAppWithFullTrustProcess/UwpAppWithFullTrustProcess/MainPage.xaml.cs (hunk 2 above)
Comment (kmahone) [reply test/MUXControlsReleaseTest/UwpAppWithFullTrustProcess/UwpAppWithFullTrustProcess/MainPage.xaml.cs]: Thanks.

**Pull request merged**